package.json

```diff
@@ -95,9 +95,11 @@
     "hoist-non-react-statics": "^2.3.1",
     "lodash": "^4.17.4",
     "markdown-to-jsx": "6.6.1",
+    "moment": "^2.18.1",
     "polished": "^1.9.0",
     "prop-types": "^15.6.0",
-    "react": "^16.2.0",
+    "react": ">=16.2.0 <16.3.0",
```

Comment:
That's good. As long as enzyme doesn't catch up, we might not be able to update here.